### PR TITLE
Improve Orchestrator

### PR DESCRIPTION
**Pull request by dillonalaird**
- Improve prompts related to fine tuning usage
- Add ability to pass `test_multi_plan` and `customized_tool_names` from VisionAgent to VisionAgentCoder

### CODE REVIEW

File: vision_agent/agent/vision_agent.py [hunk 1]
```diff
@@ -184,6 +193,12 @@ def chat_with_code(
                 or if it contains media files, it should be in the format of:
                 [{"role": "user", "content": "describe your task here...", "media": ["image1.jpg", "image2.jpg"]}]
             artifacts (Optional[Artifacts]): The artifacts to use in the task.
+            test_multi_plan (bool): If True, it will test tools for multiple plans and
+                pick the best one based off of the tool results. If False, it will go
+                with the first plan.
+            customized_tool_names (List[str]): A list of customized tools for agent to
```
Comment (shankar-vision-eng) [inline vision_agent/agent/vision_agent.py:199]: Is this similar to what @MingruiZhang developed in the UI. Send a custom list of tools to use from the overall set of tools ?

**Review by shankar-vision-eng — COMMENTED** [review]
I'm lacking context for some of the other changes. I will let @MingruiZhang review and approve

File: vision_agent/tools/tools.py [hunk 2]
```diff
@@ -1945,15 +1945,5 @@ def overlay_counting_results(
...
-        load_image,
-        save_image,
-        save_video,
-        overlay_bounding_boxes,
-        overlay_segmentation_masks,
-        overlay_heat_map,
-        overlay_counting_results,
-    ]
-)
+TOOLS_INFO = get_tools_info(TOOLS)  # type: ignore
```
Comment (MingruiZhang) [inline vision_agent/tools/tools.py:1948]: TOOLS_INFO should only expose FUNCTION_TOOLS . this line should be removed

**Review by MingruiZhang — APPROVED** [review]
LGTM, One comment to fix

File: vision_agent/agent/vision_agent.py (hunk 1 above)
Comment (dillonalaird) [reply vision_agent/agent/vision_agent.py:199]: Yes this is the same thing. It was added to VisionAgentCoder and I just made sure it worked in VisionAgent.

**Pull request merged**